src/components/IconTable.tsx

```diff
@@ -1,5 +1,5 @@
+import { Box, CSS, Flex, Icon, Tooltip } from '@atom-learning/components'
```

Comment:
What's the difference between `CSS` and `css` used in `codeblock` line 9?

Reply:
`css` is a function, `CSS` is a type

Reply:
thanks, I was wondering